byron-proxy/cfg/logging.yaml

```diff
@@ -0,0 +1,40 @@
+# global filter; messages must have at least this severity to pass:
+minSeverity: Debug
+
+# these backends are initialized:
+setupBackends:
+  - EKGViewBK
+  - KatipBK
+
+# if not indicated otherwise, then messages are passed to these backends:
+defaultBackends:
+  - EKGViewBK
+  - KatipBK
+
+# if wanted, the GUI is listening on this port:
+# hasGUI: 18321
+
+# if wanted, the EKG interface is listening on this port:
+hasEKG: 12789
+
+# if wanted, the Prometheus interface is using this port:
+hasPrometheus: 12799
```

Comment:
Will the program crash if prometheus or ekg is not available at the prescribed port? Can the host be given as well?

Reply:
@avieth that's not how prometheus works. This launches a webserver on that port with the metrics published to `/metrics` on that port. I've just tested and it's working as expected!

Reply:
EKG is the same, it's a web interface to view the same statistics. This just gives us an easier way to ingest the metrics into grafana.